Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -19,324 +19,25 @@ Create the desired types of storage in the provider Ceph cluster:
 * [RBD pools](https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool)
 * [CephFS filesystem](https://docs.ceph.com/en/quincy/cephfs/createfs/)
 
-## Commands on the source Ceph cluster
+## Connect the external Ceph Provider cluster to the Rook consumer cluster
 
-In order to configure an external Ceph cluster with Rook, we need to extract some information in order to connect to that cluster.
+1) [Export config from the Provider Ceph cluster](/Documentation/CRDs/Cluster/external-cluster/provider-export.md). Configuration must be exported by the Ceph admin, such as a Ceph keyring and mon endpoints, that allows connection to the Ceph cluster.
 
-### 1. Create all users and keys
+2) [Import config to the Rook consumer cluster](/Documentation/CRDs/Cluster/external-cluster/consumer-import.md). The configuration exported from the Ceph cluster is imported to Rook to provide the needed connection details.
+[Verify the cluster connection](/Documentation/CRDs/Cluster/external-cluster/consumer-import.md#cluster-verification) steps are provided to validate the connection.
```

Comment:
I'm still confused by mentioning the verification here. Can we just remove this line, and they can see the verification in the import topic?